code/script.js

```diff
@@ -1,67 +1,273 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
-const nameInput = document.getElementById('name-input')
-const submitBtn = document.getElementById('send-btn')     // we created the variables to work with it.
-const nameForm = document.getElementById('name-form')
-const inputWrapper = document.getElementById('input-wrapper')
+document.addEventListener('DOMContentLoaded', ()=> {
+  // Variables that point to selected DOM elements
+  const chat = document.getElementById('chat');
+  const nameInput = document.getElementById('name-input')
+  const submitBtn = document.getElementById('send-btn')     // we created the variables to work with it.
+  const nameForm = document.getElementById('name-form')
+  const inputWrapper = document.getElementById('input-wrapper')
+  
```

Comment:
// if you would like to add a sound effect see below: 
const messageSound = () => {
  let audio = new Audio("https://notificationsounds.com/storage/sounds/file-sounds-1233-elegant.mp3")
  audio.play();
}
// the link is replaceable, add this function into your const showMessage part as it is the body part that shows all the messages